docs/services/sensors.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Sensors Service"
+linkTitle: "Sensors"
+weight: 70
+type: "docs"
+description: "The Sensors Service is a built-in service that provides a central interface to all of your robot's sensors."
+tags: ["sensor", "services"]
+icon: "/services/img/icons/sensor.svg"
+images: ["/services/img/icons/sensor.svg"]
+# SME: Cheuk
+---
+
+The Sensors Service is a built-in service that provides a central interface to all of your robot's [sensors](/components/sensor), regardless of the sensor model.
+With it you can obtain readings from multiple of your robot's sensors at once.
+
+## API
+
+The Sensors Service supports the following methods:
+
+Method Name | Description
+----------- | -----------
+[`Sensors`](#sensors) | Returns a list of names of the available sensors.
+[`Readings`](#readings) | Returns a list containing each of the readings from a given list of sensors.
+
+{{% alert title="Note" color="note" %}}
+
+The following code examples assume that you have a robot configured with a gripper, and that you add the required code to connect to your robot and import any required packages at the top of your code file.
+Go to your robot's **Code Sample** tab on the [Viam app](https://app.viam.com) for boilerplate code to connect to your robot.
+
+{{% /alert %}}
+
+### Sensors
+
+Returns a list containing the `name` of each available sensor.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` (Mapping[str, Any]) (*optional*): A generic struct, containing extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- `sensors` ([ResourceName](https://python.viam.dev/autoapi/viam/gen/common/v1/common_pb2/index.html#viam.gen.common.v1.common_pb2.ResourceName)): Names of the available sensors of the robot.
```

Comment:
```suggestion
- (List[[ResourceName](https://python.viam.dev/autoapi/viam/gen/common/v1/common_pb2/index.html#viam.gen.common.v1.common_pb2.ResourceName)]): Names of the available sensors of the robot.
```
I think we're eliminating these names since they don't actually exist in RDK.
Make clear that this is a list of them (though not sure if my suggestion renders right without testing).